packages/edit-site/src/components/navigation-sidebar/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import NavigationPanel from './navigation-panel';
+import NavigationToggle from './navigation-toggle';
+
+const NavigationSidebar = ( { content, isOpen, onNavigationToggle } ) => {
+	return (
+		<>
+			<NavigationToggle
+				isOpen={ isOpen }
+				onClick={ onNavigationToggle }
+			/>
+			{ content === 'navigation' && isOpen && <NavigationPanel /> }
+		</>
```

Comment:
Should we keep the `MainDashboardButton.Slot` removed from `packages/edit-site/src/components/header/index.js`?

Reply:
I think it should now wrap `Back to Dashboard` button, instead of the main one that toggles the sidebar. That's what it does in post editor too.